release/kata-deploy-binaries.sh

```diff
@@ -112,6 +112,14 @@ install_qemu() {
 	tar xf kata-qemu-static.tar.gz -C "${destdir}"
 }
 
+# Install static firecracker asset
+install_firecracker() {
+	info "build static firecracker"
+	"${script_dir}/../static-build/firecracker/build-static-firecracker.sh"
+	info "Install static firecracker"
+	cp firecracker-static -C "${destdir}/firecracker"
```

Comment:
the statics tarball is putting the binaries on `${destdir}/opt/kata/bin/`
please add some something similar to
```
mkdir -p "${destdir}/opt/kata/bin/"
cp firecracker-static -C "${destdir}/firecracker"
```

Reply:
@jcvenegas addressed that comment. Can you please re-review.